src/converter/openapi/src/commonMain/kotlin/community/flock/wirespec/openapi/v3/OpenAPIV3Parser.kt

```diff
@@ -492,7 +492,7 @@ object OpenAPIV3Parser {
             when (val items = schema.items) {
                 is ReferenceObject -> toReference(items, schema.nullable ?: false).toIterable(isNullable)
                 is SchemaObject -> toReference(items, schema.nullable ?: false, name).toIterable(isNullable)
-                null -> TODO()
+                null -> error("items cannot be null when type is array: $name")
```

Comment:
Can we make this even more descriptive? This is an error an end-user might see as well when they supply an invalid openapi spec. Including the object  in which the `items` prop is missing in the error might be useful to them.


```suggestion
                null -> error("property 'items' of '$name' cannot be null when 'type' is array: $schema )
```


Does this work, without schema stringify blowing up?